app/assets/stylesheets/local/footer.scss

```diff
@@ -89,45 +89,44 @@
 
 }
 
-$accept-cookies-banner-tall: 5rem;
-$accept-cookies-banner-reg:  3rem;
 nav.accept-cookies-banner-nav {
-  border-top: 2px solid lightgray;
+  border-top: 3px solid #dee2e6;
```

Comment:
this is still a a literal copy of the brand color. We have our brand colors in SCSS variables. can you use the existing SCSS variable as previously suggested?

Reply:
Yeah, of course, stay tuned...